### PR TITLE
bump some deps

### PR DESCRIPTION
**Pull request by gre**

### CODE REVIEW

File: src/bridge/types.js [hunk 1]
```diff
@@ -79,10 +79,10 @@ export interface WalletBridge<Transaction> {
   isValidTransaction(account: Account, transaction: Transaction): boolean;
 
   // render the whole Fees section of the form
-  EditFees?: React$ComponentType<EditProps<Transaction>>;
+  EditFees?: *; // React$ComponentType<EditProps<Transaction>>;
```
Comment (meriadec) [inline src/bridge/types.js]: pretty sure that flow will remove that semicolon

File: src/reducers/accounts.js [hunk 2]
```diff
@@ -96,7 +96,7 @@ export function serializeAccounts(accounts: any): Account[] {
   return accounts ? accounts.map(accountModel.decode) : []
 }
 
-export function deserializeAccounts(accounts: Account[]) {
+export function deserializeAccounts(accounts: Account[]): Array<any> {
```
Comment (meriadec) [inline src/reducers/accounts.js]: I think now we don't send<->receive Accounts types anymore (we always encode them to AccountRaw) so we should be able to keep the type and remove the horrible thing here:

⏬

File: src/bridge/types.js (hunk 1 above)
Comment (gre) [reply src/bridge/types.js]: not in a type def I think

File: src/reducers/accounts.js (hunk 2 above)
Comment (gre) [reply src/reducers/accounts.js]: yeah that'(s fixed in another PR :D

**Pull request merged**